kubernetes_cluster_autoscaler/datadog_checks/kubernetes_cluster_autoscaler/__about__.py

```diff
@@ -0,0 +1,4 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+__version__ = '1.0.0'
```

Comment:
```suggestion
__version__ = '0.0.1'
```
Curious here as well: did you get this from the template or by hand?

Reply:
by hand